src/app/component/change-password/change-password.component.html

```diff
@@ -1,19 +1,19 @@
 <div class="card">
     <div class="card-body">
-      <h4 class="card-title">Change Password <a style="float: right; cursor: pointer;" (click)="onClose()" class="material-icons">clear</a></h4>
+      <h4 class="card-title">Change Password <a id="close" style="float: right; cursor: pointer;" (click)="onClose()" class="material-icons">clear</a></h4>
         <form [formGroup]="changePasswordForm">
             <mat-form-field class="example-full-width">
-                <input matInput type="password" placeholder="Current Password" formControlName="currentPassword">
+                <input id="currentPassword" matInput type="password" placeholder="Current Password" formControlName="currentPassword">
```

Comment:
@VishalMajumdar09 we use camel casing in JS/TS as standard for defining HTML ids please use hyphens.

for example: currentPassword should be current-password, sendMessageBtn should be send-message-btn

please update the similar way in all the places required

Reply:
I have done the changes as per your comments.